package/sap-installation-wizard.changes

```diff
@@ -1,7 +1,11 @@
 -------------------------------------------------------------------
-Tue Sep 10 11:13:32 UTC 2024 - Peter Varkoly <varkoly@suse.com>
+Mon Feb 24 06:57:04 UTC 2025 - Peter Varkoly <varkoly@suse.com>
```

Comment:
I am not sure if OBS/reviewers willbe happy with that retrospective change of changes, but I am not against.

Reply:
Thanks :-)